src/main/scala/viper/server/ViperConfig.scala

```diff
@@ -83,4 +101,29 @@ class ViperConfig(args: Seq[String]) extends ScallopConf(args) {
     noshort = false,
     hidden = false
   )
-}
\ No newline at end of file
+
+  val nThreads: ScallopOption[Int] = opt[Int](
+    name = "nThreads",
+    descr = s"Maximal number of threads that should be used (not taking threads used by backend into account)\n" +
+      s"Values below ${DefaultVerificationExecutionContext.minNumberOfThreads} (the minimum) will be set to the minimum.\n" +
+      s"The default value is the maximum of ${DefaultVerificationExecutionContext.minNumberOfThreads} and the number of available processors",
```

Comment:
This seems strange. Why can I not run the server using fewer threads than the number of processors? This would seem natural e.g. on a big machine with dozens of cores.

Reply:
We need 2 or 3 threads at minimum such that there is no deadlock. The number of processors is only used to derive the default value. There seems to be the general recommendation to use as many threads as processors but we need at least the minimal amount

Reply:
But from your explanation it doesn't follow that running ViperServer on a machine with 64 cores with "only" 32 threads is per se wrong. How about setting the minimum to 3 and not having a maximum?

Reply:
I think there is some confusion on your side. There is no maximum. The max is only used to calculate the default value